test/docfx.Test/build/ConfigTest.cs

```diff
@@ -1,12 +1,25 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections;
+using Newtonsoft.Json;
 using Xunit;
 
 namespace Microsoft.Docs.Build;
 
 public static class ConfigTest
 {
+    [Theory]
+    [InlineData("DOCFX_FOO", "bar", "{'foo':'bar'}")]
+    [InlineData("DOCFX_FOO", "a;b;c", "{'foo':['a','b','c']}")]
```

Comment:
How about input the value as Json array? ['a', 'b', 'c'] in this case.
We can simply use `JToken.Parse(value)`

Reply:
This is the existing behavior. `;` is simpler than writing a proper json array xD